src/components/ErrorMessage/ErrorMessage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { string } from 'prop-types';
+
+export const ErrorMessage = (props) => {
+  const { message } = props;
+  return (<div>{message}</div>);
+};
```

Comment:
I think you could allow a `style` prop to be passed to the Error Message Component. Depending on the page it is being used, we might want to add some styling.

Reply:
done